src/app/profile-setting/profile-setting.component.ts

```diff
@@ -56,13 +55,6 @@ export class ProfileSettingComponent implements OnInit, OnDestroy {
                private location: Location,
                private modalService: ModalService,
                private nativeNotificationService: NativeNotificationService) {
-    this.subscriptions.push(store.select('user').subscribe((user: User) => {
```

Comment:
If we remove this I think we lose user identicon, hmm?

Reply:
we should discuss this because it is a little bit wired